ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -337,3 +337,38 @@ def test_jax_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="jax.numpy.where",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric")),
```

Comment:
for the function `helpers.dtype_and_values` you should also add the argument.
`min_num_dims=1` so that it doesn't produce 0d arrays

Reply:
Thanks, I have updated it!